### PR TITLE
Select / deselect all meta fields

### LINKED ISSUE #146: Enable / Disable all fields
In response to https://wordpress.org/support/topic/feature-suggestion-select-all-in-settings/ it would be very useful to be able to enable / disable all fields at once.

This includes:

- The checkboxes on the *Default Enabled Meta Fields* section of the global settings
- The drag & drop selection per form
- The drag & drop section the feed add-ons

### PR DESCRIPTION
**Pull request by doekenorg**
This PR resolves #146

Once merged this should be ported to the v2 feed version of the plugin.

- [x] Select all / Deselect all checkbox on meta
- [x] Enable / Disable all fields on sortable fields
- [x] Translate Enable all / Disable all in javascript using json blob
- [x] Fix styling of the buttons via inline to force look in lite & pro

### CODE REVIEW

File: src/GFExcelAdmin.php [hunk 1]
```diff
@@ -1097,7 +1097,25 @@ public function scripts()
                     ]
                 ],
                 'deps' => ['jquery', 'jquery-ui-sortable', 'jquery-ui-datepicker'],
+                'callback' => function() {
+                    // Can't use `scripts` because `gfexcel-js` will break JS, and can't be renamed at this point for the add-ons.
+                    wp_localize_script('gfexcel-js', 'gfexcel_strings', [
+                        'enable' => esc_html__('Enable all', GFExcel::$slug),
```
Comment (mrcasual) [inline src/GFExcelAdmin.php:1103]: @doekenorg please replace `GFExcel::$slug` with the actual text domain.

File: src/GFExcelAdmin.php [hunk 2]
```diff
@@ -1097,7 +1097,25 @@ public function scripts()
...
+                        'disable' => esc_html__('Disable all', GFExcel::$slug),
```
Comment (mrcasual) [inline src/GFExcelAdmin.php:1104]: @doekenorg please replace GFExcel::$slug with the actual text domain.

File: src/GFExcelAdmin.php (hunk 1 above)
Comment (mrcasual) [reply src/GFExcelAdmin.php:1103]: @doekenorg, if possible, please replace everywhere in the code.

**Comment by rafaehlers** [comment]
- [ ] The `Date Updated` meta doesn't appear on the Forms > Settings > GravityExport Lite > Default Enabled Meta Fields

image

- [ ] But it does appear already pre-added on the specific form settings:

image

Everything else looks good and working fine 👍

**Comment by zackkatz** [comment]
@doekenorg Can you please resolve the merge conflicts?

**Comment by doekenorg** [comment]
@zackkatz haha, easier said than done, because the base is completely different now. But I got it working!

**Pull request merged**